rootfs/usr/share/opengamepadui/scripts/powertools

```diff
@@ -8,65 +8,79 @@ fi
 
 ## CPU Generics
 cpuBoost(){
-  /usr/bin/echo $1 > /sys/devices/system/cpu/cpufreq/boost
-}
-
-cpuPower(){
-  /usr/bin/cpupower $1 $2 $3
+ state=$1
+  /usr/bin/echo $state > /sys/devices/system/cpu/cpufreq/boost
```

Comment:
:tada:

Reply:
?

Reply:
Just a fan of using variable names.

Reply:
Yeah, I conceded after I had to remember what each of these did specifically.